### PR TITLE
Added dbt-azuresynapse

### PR DESCRIPTION
**Pull request by mattberns**
## Description & motivation
Added a reference to [dbt-azuresynapse](https://github.com/embold-health/dbt-azuresynapse) a community contributed adapter for Azure Synapse DW an azure big-data-warehouse offering.

## Edited
- website/docs/docs/supported-databases.md

## Added
- website/docs/reference/warehouse-profiles/azuresynapse-profile.md

### CODE REVIEW

**Review by jtcohen6 — APPROVED** [review]
Very nice! Thanks for documenting your plugin @mattberns :)

I see in the `dbt-azuresynapse` repo that you're using `dbt-integration-tests`. We just released a new adapter testing framework, I'll open an issue in the repo with details.

File: website/docs/reference/warehouse-profiles/azuresynapse-profile.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Configure your dbt profile for using SQL Server authentication or Integrated Security:
+
+##### SQL Server authentication
+```yaml
+dbt-azuresynapse:
+  target: dev
+  outputs:
+    dev:
+      type: sqlserver
```
Comment (jtcohen6) [inline website/docs/reference/warehouse-profiles/azuresynapse-profile.md]: Just caught this, I think you want this to be
```suggestion
      type: azuresynapse
```

**Comment by mattberns** [comment]
@jtcohen6 Good catch above. I have committed your suggestion.

**Comment by jtcohen6** [comment]
@mattberns Last thing: could you add the new file `reference/warehouse-profiles/azuresynapse-profile` to [this list](https://github.com/fishtown-analytics/docs.getdbt.com/blob/current/website/sidebars.js#L436-L440), so that it will show up on the sidebar?

**Comment by mattberns** [comment]
@jtcohen6 Jeremy when is the next release of the docs to the production site? It would be good to have a reference for future PRs if I add to docs here. Is there an update cadence for `current` branch?

**Comment by jtcohen6** [comment]
@mattberns The production site is continuously deployed from the `current` branch. Check it out: https://docs.getdbt.com/reference/warehouse-profiles/azuresynapse-profile

**Comment by mattberns** [comment]
> @mattberns The production site is continuously deployed from the `current` branch. Check it out: https://docs.getdbt.com/reference/warehouse-profiles/azuresynapse-profile

Ah! Had to clear my cache!

**Pull request merged**